### PR TITLE
Bug 1215562: robots.txt file

### PR DESCRIPTION
**Pull request by EmiXLabs**
Tested locally, using sslserver (https://github.com/teddziuba/django-sslserver) module in local.py, works.

### CODE REVIEW

**Comment by Osmose** [comment]
Thanks for the PR!

File: pontoon/urls.py [hunk 1]
```diff
@@ -40,6 +41,9 @@
     url(r'^pontoon\.js$',
         RedirectView.as_view(url='/static/js/pontoon.js', permanent=True)),
 
+    # Robots.txt
+    (r'^robots.txt$', lambda r: HttpResponse("User-agent: *\nDisallow: /a/\nDisallow: /admin/", content_type="text/plain")),
```
Comment (Osmose) [inline pontoon/urls.py]: This totally works, but I'd rather have this in a template file for maintainability's sake. A simple view that just calls render should be all that's needed.

Comment (EmiXLabs) [inline pontoon/urls.py]: @Osmose Added in one line as it will not be used a lot, but yes, I can change it to template if needed.

Comment (EmiXLabs) [inline pontoon/urls.py]: BTW, @Osmose, which one will be better, rendering in views or using direct_to_template in urls? The only extra line in views will be logging I guess.

**Comment by mathjazz** [comment]
Looks good @EmiXLabs! Mind rebasing into a single commit?

File: pontoon/urls.py (hunk 1 above)
Comment (Osmose) [inline pontoon/urls.py]: You seem to have already found a fine answer to that question. :P
For the record, the in-view logging isn't something we want to keep doing, we just haven't cleaned it up yet. So no worries.

**Comment by mathjazz** [comment]
@Osmose says r+ on IRC!

Congrats on your first PR merged, @EmiXLabs!

**Pull request merged**